common/wireguard-types/Cargo.toml

```diff
@@ -13,13 +13,16 @@ license.workspace = true
 [dependencies]
 base64 = { workspace = true }
 dashmap = { workspace = true }
+futures = { workspace = true }
 log = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
 thiserror = { workspace = true }
+tokio = { workspace = true, features = ["rt-multi-thread", "net", "signal"] } # async runtime
```

Comment:
I'm just judging based on the crate name(I haven't reviewed this PR, just looked at CI errors) - this import probably causes the wasm-sdk to fail to build since it can't use tokio runtime.
You might probably want to leave this crate with types only and move anything extra to another crate

Reply:
Removing the features was enough to make it build for wasm